src/niscope/system_tests/test_system_niscope.py

```diff
@@ -195,6 +195,34 @@ def test_fetch_double_into(session):
             assert record_wfm[j] == waveform[i * test_record_length + j]
 
 
+def test_get_self_cal_last_date_time(session):
+    last_cal = session.get_self_cal_last_date_and_time()
+    assert last_cal.month == 12
+    assert last_cal.day == 21
+    assert last_cal.year == 1999
+    assert last_cal.hour == 0
+    assert last_cal.minute == 0
+
+
+def test_get_ext_cal_last_date_time(session):
+    last_cal = session.get_ext_cal_last_date_and_time()
+    assert last_cal.month == 12
+    assert last_cal.day == 21
+    assert last_cal.year == 1999
+    assert last_cal.hour == 0
+    assert last_cal.minute == 0
+
+
+def test_get_self_cal_last_temperature(session):
+    last_cal_temp = session.get_self_cal_last_temp()
+    assert last_cal_temp == 25
+
+
+def test_get_ext_cal_last_temperature(session):
+    last_cal_temp = session.get_ext_cal_last_temp()
+    assert last_cal_temp == 25
+
+
 def test_self_test(session):
     # We should not get an assert if self_test passes
     session.self_test()
```

Comment:
where is the test for niScope_CalFetchMiscInfo ?
Check out codecov results it shows how it is not covered.

Reply:
Removed altogether for the time being. I can add this separately as part of solving #1464 in order to ensure the additions are consistent.